volt.gemspec

```diff
@@ -7,10 +6,15 @@ version = File.read(File.expand_path('../VERSION', __FILE__)).strip
 Gem::Specification.new do |spec|
   spec.name          = 'volt'
   spec.version       = version
+  spec.platform      = Gem::Platform::RUBY
+  spec.required_ruby_version = '>= 2.1'
```

Comment:
Here we say the problem with aligning code on `=` - one line can potentially force a change in indentation of every other line around it.